crypto/chacha/asm/chacha-x86_64.pl

```diff
@@ -440,13 +442,14 @@ sub SSSE3ROUND {	# critical path is 20 "SIMD ticks" per round
 	&por	($b,$t);
 }
 
-my $xframe = $win64 ? 32+32+8 : 24;
+my $xframe = $win64 ? 32+8 : 8;
```

Comment:
+8 is "tribute" to the fact that ABI says "top of stack should be 16-byte aligned". Upon entry to function it's misaligned because return address is pushed. In other words, it's padding, and yes, nothing is stored there.